CRITICAL - 2020-12-13 21:10:42 --> Unable to connect to the database.
#0 C:\xampp\htdocs\ohayo\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\ohayo\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1952): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#2 C:\xampp\htdocs\ohayo\vendor\codeigniter4\framework\system\Model.php(1673): CodeIgniter\Database\BaseBuilder->countAllResults(false)
#3 C:\xampp\htdocs\ohayo\vendor\codeigniter4\framework\system\Model.php(1176): CodeIgniter\Model->countAllResults(false)
#4 C:\xampp\htdocs\ohayo\app\Controllers\Home.php(23): CodeIgniter\Model->paginate(3, 'artikel')
#5 C:\xampp\htdocs\ohayo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Home->index()
#6 C:\xampp\htdocs\ohayo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\ohayo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ohayo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ohayo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#10 {main}
